### ISSUE TITLE
modify `skeleton-pypi` to add dependencies

### ISSUE THREAD

**Issue by cekees** [hashdist/hashdist#338]
Currently the `skeleton-pypi`  command doesn't attempt to deal with dependencies of pypi packages, but I  think it  could.

**Comment by certik** [comment]
Yeah --- it should add all dependencies into `dependencies` and also create the yaml files for dependencies, unless they already exist. To figure out if the given dependency `PACKAGE` exists, it should look for `pkgs/PACKAGE.yaml` as well as `pkgs/python-PACKAGE.yaml`.

Another question is about how to name packages from pypi, and I created a separate issue for that (https://github.com/hashdist/hashstack/issues/795).

**Comment by jcftang** [comment]
I think this is a good idea, so +1
